package/MDAnalysis/topology/TOPParser.py

```diff
@@ -620,3 +671,23 @@ def parse_dihedrals(self, diha, dihh):
         dihedrals = Dihedrals(dihed)
         impropers = Impropers(improp)
         return dihedrals, impropers
+
+    def parse_chainids(self, num_per_record, numlines):
```

Comment:
We're slowly adding typing, could you add types here please?

Reply:
Done.